modules/swagger-codegen/src/test/java/io/swagger/codegen/swift3/Swift3CodegenTest.java

```diff
@@ -14,13 +14,13 @@
     Swift3Codegen swiftCodegen = new Swift3Codegen();
 
     @Test
-    public void testReservedWord() throws Exception {
-	Assert.assertEquals(swiftCodegen.toEnumVarName("Public", null), "_public");
+    public void testCapitalizedReservedWord() throws Exception {
+      Assert.assertEquals(swiftCodegen.toEnumVarName("AS", null), "_as");
```

Comment:
@gierschv I would suggest using 4-space instead of 2-space for indentation to make it consistent with the rest of the file.